<launch>
    <!--Simulator-->
    <include file="$(find very_simple_robot_simulator)/launch/minimal_simulator_py.launch"/>
    <include file="$(find very_simple_robot_simulator)/launch/openni_simulator_py.launch"/>
    <include file="$(find very_simple_robot_simulator)/launch/world_state.launch"/>

    <!--PID Controller-->
    <node name="angle_controller" pkg="pid" type="controller" ns="pathline_follower">
        <param name="Kp" value="0.01" />
        <param name="Ki" value="0.0" />
        <param name="Kd" value="0.0" />
        <param name="upper_limit" value="0.5" />
        <param name="lower_limit" value="-0.5" />
    </node>

    <!--Nodes-->
    <node pkg="low_level_control" name="follow_the_carrot" type="follow_the_carrot.py"/>
    <!--
    <node pkg="low_level_control" name="pathline_reader" type="pathline_reader.py"/>
    -->
</launch>
